packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -751,6 +754,7 @@ class RefreshProgressIndicator extends CircularProgressIndicator {
     double strokeWidth = defaultStrokeWidth, // Different default than CircularProgressIndicator.
     String? semanticsLabel,
     String? semanticsValue,
+    this.additionalArrowheadScale = 1.0,
```

Comment:
Same here.

Reply:
Here it seems to not be used anywhere else, and the value is trivial - do we really need this?

Reply:
You are right, I think we can leave it as is.